site/zenodo_rdm/moderation/handlers.py

```diff
@@ -73,14 +73,17 @@ def rules(self):
         """Get scoring rules."""
         if isinstance(self._rules, str):
             return current_app.config[self._rules]
-        return self._rules or []
+        return self._rules or {}
+
+    def evaluate_result(self, params):
+        return sum(params.values())
```

Comment:
We'll change the `evaluate_result` once we have the spam rule in, so that it only sums the values that make sense as part of the score, and uses the spam probability separately "decision-tree-style".